app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -94,6 +94,10 @@
 
     vm.search = dimSearchService;
 
+    $scope.$watch('vm.search.query', function() {
```

Comment:
(just curious) is it better to use this watcher here versus listening to the ng-change event?